examples/js/loaders/GLTFLoader.js

```diff
@@ -4,6 +4,7 @@
  * @author Tony Parisi / http://www.tonyparisi.com/
  * @author Takahiro / https://github.com/takahirox
  * @author Don McCurdy / https://www.donmccurdy.com
+ * @author Timoshenko Konstantin / https://github.com/eXponenta
```

Comment:
The authorship of files is normally not changed if you just make a minor addition to a file.